testing/endtoend/components/validator.go

```diff
@@ -187,6 +187,12 @@ func (v *ValidatorNode) Start(ctx context.Context) error {
 		beaconRPCPort = e2e.TestParams.Ports.PrysmBeaconNodeRPCPort
 	}
 
+	beaconRestApiPort := e2e.TestParams.Ports.PrysmBeaconNodeGatewayPort + index
+	if beaconRestApiPort >= e2e.TestParams.Ports.PrysmBeaconNodeGatewayPort+e2e.TestParams.BeaconNodeCount {
```

Comment:
Remove this check here, we should always have validators pointing to the correctly allocated port. If the beacon api port leads to some non-existent being used, its better to fail early rather than invisibly.